19 - Webcam Fun/scripts-FINISHED.js

```diff
@@ -8,7 +8,13 @@ function getVideo() {
   navigator.mediaDevices.getUserMedia({ video: true, audio: false })
     .then(localMediaStream => {
       console.log(localMediaStream);
-      video.src = window.URL.createObjectURL(localMediaStream);
+    
+//     DEPRECIATION : 
+//       The following has been depreceated by major browsers.
+//       video.src = window.URL.createObjectURL(localMediaStream);
+//       Please refer to this:
```

Comment:
Hi, thanks for your contribution. But I think you forgot to add the link in comments which one can refer to for further information about createObjectURL depreciation.

Regards!